static/app/components/events/interfaces/threads.spec.tsx

```diff
@@ -39,6 +40,8 @@ describe('Threads', function () {
     });
     ProjectsStore.loadInitialData([project]);
     ConfigStore.set('user', UserFixture());
+
+    localStorage.setItem('issue-details-stracktrace-display', JSON.stringify([]));
```

Comment:
whats this do

Reply:
some of the tests were failing because the one test would select a display option which another test wasn't expecting, this just resets the state so the tests are independent